### DISCUSSION TITLE
Input field

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3216] [Office hours] [answered]
### Background

Hey Paste team,

The product I'm working on is Mappings, customers use this to customize how they want to send their data to downstream tools.

Here's a loom walking through the feature background + existing interaction: https://www.loom.com/share/eab3548456b74d22bc30b273e4337b5f

### Feedback

I'd love to get your thoughts on how we might Paste-ify the mapping UI interaction in Segment as I don't think we have an existing paste component that handles this type of use case

### Stage of design

Exploration

### Are you looking for visual design support?

- [X] Yes

**Comment by nkrantz** [comment, marked as answer]
Hi @char-mainewang - following up with Office Hours from this past Thurs. [If you'd like to review the recording, it can be found here ](https://www.loom.com/share/2f28039a33104da2afb29f93978148f7)(password in #help-design-systems slack). Please keep us updated with your progress on the migration of the Mappings UI! Thanks for attending.
